boltdesignsystem/bolt
# DS-193: fix list wrapping bug in safari

## Jira

https://pegadigitalit.atlassian.net/browse/DS-193

## Summary

Fixes a bug where inline list pushes its last list item to the next line even if there's enough space to fit all items.

## Details

1. Safari specific CSS added to fix the wrapping issue.
2. CSS is commented stating it's needed for Safari.

## How to test

Run the branch locally and check the list docs. Also, view this [band doc](http://localhost:3000/pattern-lab/?p=components-band-with-pinned-content) and make sure the share tool's last item doesn't get pushed to the next line. It should be visually the same across Safari, Chrome, and Firefox.
